src/util/uint.rs

```diff
@@ -26,7 +26,7 @@ use util::BitArray;
 macro_rules! construct_uint {
     ($name:ident, $n_words:expr) => (
         /// Little-endian large integer type
-        #[repr(C)]
```

Comment:
Why you have removed `#[repr(C)]`? It may break some code relying on rust-bitcoin through FFI...

Reply:
because it makes no sense to me to use this struct in FFI.
it's a uint with 64bit limbs little endian. it's very specific to the implementation here.

Reply:
Than we need to remember to do a major version bump after this PR merge into the master

Reply:
yep. see breaking changes no. 2 in the post :)